target/linux/ar71xx/image/tiny-legacy-devices.mk

```diff
@@ -152,3 +152,10 @@ define LegacyDevice/NBG_460N_550N_550NH
   DEVICE_PACKAGES := kmod-rtc-pcf8563
 endef
 LEGACY_DEVICES += NBG_460N_550N_550NH
+
+define LegacyDevice/F9K1115V2
+  DEVICE_TITLE := Belkin AC1750DB (F9K1115V2)
+  DEVICE_PACKAGES := kmod-usb-core kmod-usb2 kmod-usb3 kmod-usb-ledtrig-usbport \
+        kmod-ath10k ath10k-firmware-qca988x
+endef
+LEGACY_DEVICES += F9K1115V2
```

Comment:
this is also not ordered correctly

Reply:
Hi,

I moved belkin to the top. Existing entries were not ordered alphabetically based on device name or device title. Let me know where you would like me to place the entry for f9k1115v2.